tritium-lib/src/main/java/com/palantir/tritium/proxy/Instrumentation.java

```diff
@@ -118,41 +108,32 @@ private Builder(Class<T> interfaceClass, U delegate) {
             this.delegate = checkNotNull(delegate, "delegate");
         }
 
-        /**
-         * Supply additional metrics name prefix to be used across interfaces that use MetricGroup annotations.
-         *
-         * Example:
-         *
-         * Given a prefix com.business.service and instrumented interfaces below, a single metric
-         * "com.business.service.fastcall" will share recordings across classes.  Functionality assumes a shared
-         * MetricsRegistry.
-         *
-         * interface WidgetService {
-         *     @MetricGroup("fastcall")
-         *     getWidgets();
-         * }
-         *
-         * interface UserService {
-         *     @MetricGroup("fastcall")
-         *     getUsers();
```

Comment:
@pdunn : I'm removing the annotation processing stuff for now as we can now explicitly set tags via MetricName, and I think we could support converting annotations into tags via the `ServiceMetricNameFunction`.